lib/grape-rabl/render.rb

```diff
@@ -1,8 +1,8 @@
 module GrapeRabl
   module Render
     def render(options = {})
-      env['api.endpoint'].options[:route_options][:rabl] = options.delete(:rabl) if options.include?(:rabl)
-      env['api.endpoint'].options[:route_options][:rabl_locals] = options.delete(:locals)
+      env['api.tilt.rabl'] = options.delete(:rabl) if options.include?(:rabl)
```

Comment:
`key?` and `include?` are all aliases of `has_key?`, is there any technical reason use `key?` instead of `include?`?